### ISSUE TITLE
[Accessibility Audit] Code Intel: Inspect auto-indexing jobs for an instance (site-admin)

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#33515]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1kA6aVOAgID_uPm-d6uEC1DG6WHjJ8fPT0UAXQcML_KQ/edit?usp=sharing). Use this for further context.

- Navigate to site admin settings in Sourcegraph.
- Click on Auto-indexing in the 🧠 Code Intelligence section in the sidebar.
- Rest of the steps are the same as for [Inspecting auto-indexing jobs for a repository](https://docs.google.com/document/d/1kA6aVOAgID_uPm-d6uEC1DG6WHjJ8fPT0UAXQcML_KQ/edit#heading=h.nokk92k9dc2)

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by sourcegraph-bot-2** [comment]
Heads up @macraig - the "team/code-intelligence" label was applied to this issue.

**Comment by efritz** [comment]
Description of journey with screen brightness down:

- [x] Noah was able to open the user settings dropdown fine
- [ ] Announces "three items `Signed in as`, `@`, `Strum355` and 14 more items" on open. Could do a better job of saying what's available.
- [ ] Very confused once the dropdown was open; difficult to navigate into the menu. Seems to require the user to move out and then back in to get into the dropdown content.
- [x] Once navigation entered the user settings dropdown content navigation went as expected
- [ ] (Existing issue) navigation in sidebar does not announce any content change
- [ ] Pop-up menus don't list (on firefox) what is currently selected without navigation. Very confusing experience.
- [ ] General concern: loading content is not announced so users navigating to a loading filter view that takes 30s to render might be a very confusing experience (difficult to tell what that experience would be in a concrete way though)

The remaining issues should be the same as https://github.com/sourcegraph/sourcegraph/issues/33513.

**Comment by varungandhi-src** [comment]
Closing this since all issues have been filed, so this audit has been completed.
